### ISSUE TITLE
fix(docs): include scope parameter in login method documentation

### ISSUE THREAD

**Issue by QuentinBrosse** [prescottprue/react-redux-firebase#380]
Is it possible to specify [additional OAuth 2.0 scopes](https://developers.facebook.com/docs/facebook-login/permissions) in the [login function](http://react-redux-firebase.com/docs/auth.html#logincredentials) ?

Something like this:
```js
props.firebase.login({
  provider: 'google',
  type: 'popup',
  scopes: ['email', 'public_profile', 'user_friends']
})
```

I can't find anything who speak about it in the documentation and the issues.
Thanks a lot :)

**Comment by prescottprue** [comment]
Yes, it works exactly as you indicated (passing an array or string to `scopes` parameter). The `email` scope is included by default.

I didn't realize it wasn't covered clearly in the `login` method docs. I'll update that, thanks for reaching out.

**Comment by prescottprue** [comment]
Included as part of the [`v2.0.2` release](https://github.com/prescottprue/react-redux-firebase/releases/tag/v2.0.2).

Let me know if you think anything else should be added to make it more clear.
